Source/Core/DolphinQt/Config/Graphics/GeneralWidget.cpp

```diff
@@ -247,15 +251,21 @@ void GeneralWidget::AddDescriptions()
       QT_TR_NOOP("Uses the main Dolphin window for rendering rather than "
                  "a separate render window.<br><br><dolphin_emphasis>If unsure, leave "
                  "this unchecked.</dolphin_emphasis>");
-  static const char TR_ASPECT_RATIO_DESCRIPTION[] =
-      QT_TR_NOOP("Selects which aspect ratio to use when rendering.<br>"
-                 "Each game can have a slightly different native aspect ratio."
-                 "<br><br>Auto: Uses the native aspect ratio"
-                 "<br>Force 16:9: Mimics an analog TV with a widescreen aspect ratio."
-                 "<br>Force 4:3: Mimics a standard 4:3 analog TV."
-                 "<br>Stretch to Window: Stretches the picture to the window size."
-                 "<br>Custom: For games running with specific custom aspect ratio cheats.<br><br>"
-                 "<dolphin_emphasis>If unsure, select Auto.</dolphin_emphasis>");
+  static const char TR_ASPECT_RATIO_DESCRIPTION[] = QT_TR_NOOP(
+      "Selects which aspect ratio to use when drawing on the render window.<br>"
+      "Each game can have a slightly different native aspect ratio.<br>They can vary by "
+      "scene and settings and rarely ever exactly match 4:3 or 16:9."
+      "<br><br><b>Auto</b>: Uses the native aspect ratio"
+      "<br><b>Force 16:9</b>: Mimics an analog TV with a widescreen aspect ratio."
+      "<br><b>Force 4:3</b>: Mimics a standard 4:3 analog TV."
+      "<br><b>Stretch to Window</b>: Stretches the picture to the window size."
+      "<br><b>Custom</b>: Forces a specific aspect ratio."
+      "<br>This is mostly meant to be used with matching custom aspect ratio game cheats."
+      "<br><b>Custom (Stretch)</b>: Similar to `Custom` but not relative to the "
+      "native game aspect ratio.<br>Not meant to be used under normal circumstances."
```

Comment:
I feel like "but not relative to the native game aspect ratio" could lead a user to divide the desired aspect ratio for Custom by either 4:3 or the native aspect ratio. (E.g., 64:27 is 16:9 relative to 4:3, so enter that, or perhaps it should be 17:9 for a somewhat narrower game.) A better wording might be "Similar to \`Custom\` but leads to a stretched picture in most cases, ignoring the game's native aspect ratio."

Reply:
Actually, Custom could be rewritten to "Mimics an analog TV with the specified aspect ratio.&lt;br&gt;This is mostly meant to be used with matching custom aspect ratio game cheats.", which in my opinion could allow Custom (Stretch) to have the more concise description of "Stretches the picture to exactly the specified aspect ratio.&lt;br&gt;Not meant to be used under normal circumstances." without confusion.

Reply:
https://github.com/dolphin-emu/dolphin/pull/12436 updated